### DISCUSSION TITLE
[Feature Request] Add an option to disable the loading animation for embedded Videos

### DISCUSSION THREAD

**Discussion by austen-wade** [vimeo/player.js#785] [Ideas]
Please add the option to disable the spinner animation while an embedded video is loading.

This would include:
- Creating an option in the Javascript Player options
- Creating an option via the embed URL

My use case:
Vimeo players are using a placeholder/fallback image that sits behind the player.
Currently if I want to disable the animation, I have to create a custom implementation where I listen for a play event and toggle CSS display (for a simple approach). Having an option would be a convenience for me as a developer per client requests.

Thank you

**Comment by lssamlyon** [comment]
Thanks for raising this to us, we are always looking to continue to improve the functionality. Im going to move this over to discussions and make a note internally with the team.

**Reply by sizandyola** [reply]
Is there an update to this?

**Comment by muhammadnaveed954** [comment]
Is there any update guys
